srs_env_model_msgs/PlaneArray plane_array
--- 
uint32[] object_ids